[2024-01-28T16:02:51.152+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-03-11T00:00:00+00:00 [queued]>
[2024-01-28T16:02:51.229+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-03-11T00:00:00+00:00 [queued]>
[2024-01-28T16:02:51.242+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-28T16:02:52.495+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_stackoverflow_data> on 2023-03-11 00:00:00+00:00
[2024-01-28T16:02:52.815+0000] {standard_task_runner.py:60} INFO - Started process 3539 to run task
[2024-01-28T16:02:53.151+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_stackoverflow_data', 'get_stackoverflow_data', 'scheduled__2023-03-11T00:00:00+00:00', '--job-id', '1392', '--raw', '--subdir', 'DAGS_FOLDER/loading_data_to_aws_S3.py', '--cfg-path', '/tmp/tmpqzw6cwld']
[2024-01-28T16:02:53.173+0000] {standard_task_runner.py:88} INFO - Job 1392: Subtask get_stackoverflow_data
[2024-01-28T16:02:53.908+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-03-11T00:00:00+00:00 [running]> on host 9924936b75b5
[2024-01-28T16:02:54.807+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='charlotte' AIRFLOW_CTX_DAG_ID='fetch_stackoverflow_data' AIRFLOW_CTX_TASK_ID='get_stackoverflow_data' AIRFLOW_CTX_EXECUTION_DATE='2023-03-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-03-11T00:00:00+00:00'
[2024-01-28T16:02:55.930+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-28T16:02:56.041+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=fetch_stackoverflow_data, task_id=get_stackoverflow_data, execution_date=20230311T000000, start_date=20240128T160251, end_date=20240128T160256
[2024-01-28T16:02:56.625+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-28T16:02:56.827+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
